SECURITY.md

```diff
@@ -1,9 +1,38 @@
 # Security Policy
 
+This document outlines security procedures and general policies for the `OnDemand`
+project.
+
 ## Security Audits
 
-[Trusted CI](https://trustedci.org/) , the NSF Cybersecurity Center of Excellence conducted an in-depth vulnerability assessment of Open OnDemand, completing it in December 2018. This assessment included a careful review of the code, increasing our confidence in its security. The Ohio Supercomputing Center addressed the implementation issues (bugs) that were found during this review, producing a more robust revision of Open OnDemand.
+We have used [Trusted CI](https://trustedci.org/) in the past, the NSF Cybersecurity Center of 
+Excellence conducted an in-depth vulnerability assessment of Open OnDemand, completing 
+it in December 2018. This assessment included a careful review of the code, increasing 
+our confidence in its security. The Ohio Supercomputing Center addressed the implementation 
+issues (bugs) that were found during this review, producing a more robust revision of Open OnDemand.
+
+We hope to have more audits on a regular ongoing basis and to share those results when applicable.
```

Comment:
Revert this back to the original copy Alan added and remove this line:

>We hope to have more audits on a regular ongoing basis and to share those results when applicable

Reply:
You don't want audits to become regular or results public?

Reply:
Also, Alan's line is one long line, this is broken up to be more readable in an ide/viewer.  You know, 100 column limit sort of thing.